### ISSUE TITLE
Very blurry after update

### ISSUE THREAD

**Issue by Morten242** [Microsoft/vscode#56313]
Issue Type: <b>Bug</b>

1. Open vscode

Result: It's all blurry
Expect: Not blurry

Reproduced with extensions disabled.

![blurry](https://user-images.githubusercontent.com/6974432/44078364-924487f8-9fa6-11e8-8ea1-453c1c59088d.gif)

VS Code version: Code 1.26.0 (4e9361845dc28659923a300945f84731393e210d, 2018-08-13T16:29:31.933Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.91GB (5.95GB free)|
|Screen Reader|no|
|VM|0%|

</details>

**Comment by Morten242** [comment]
Yeah, it seems to fix the issue; don't see the extreme blur in that build.

**Comment by bpasero** [comment]
@Morten242 thanks. As a workaround you can run "`code --disable-gpu`" and it should also resolve the issue with stable.
